src/oc_erchef/apps/data_collector/rebar.config

```diff
@@ -1,6 +1,22 @@
 %% -*- mode: erlang -*-
 %% -*- tab-width: 4;erlang-indent-level: 4;indent-tabs-mode: nil -*-
 %% ex: ts=4 sw=4 ft=erlang et
+
+{deps,
+    [
+    %% lager has to come first since we use its parse transform
+    {lager, ".*",
+        % try this to cure lager master verify test issue with bookshelf
+        {git, "git://github.com/erlang-lager/lager.git",                 {branch, "master"}}},
+        %{git, "git://github.com/basho/lager.git",                       {branch, "master"}}},
```

Comment:
If we are moving to erlang-lager/lager, let's clean up the commented code here.

Reply:
Will do.

Reply:
Done.